enterprise/backend/src/metabase_enterprise/semantic_search/index.clj

```diff
@@ -149,7 +160,7 @@
   "Returns the default index spec for a model."
   [embedding-model]
   (let [{:keys [model-name provider vector-dimensions]} embedding-model
-        table-name (str "index_table__" provider "__" model-name "__" vector-dimensions)]
+        table-name (str "index_table_" provider "_" model-name "_" vector-dimensions)]
```

Comment:
Had to shave these double underscores down to single underscores to get under the postgres limit for identifier names for the index on the new column. Alternatively, we could abbreviate the model names or otherwise shorten the prefix. See comment on `index-name` below.